src/psbt.h

```diff
@@ -137,6 +137,9 @@ template<typename Stream, typename... X>
 void UnserializeFromVector(Stream& s, X&... args)
 {
     size_t expected_size = ReadCompactSize(s);
+    if (!expected_size) {
+        return; /* Zero size = no data to read */
+    }
```

Comment:
I'm curious why this isn't needed anywhere upstream. What would happen without this line?

Reply:
Okay, I guess the answer is just that no other expected-to-be-empty fields are de/serialized as vectors. Alright, I'm happy.